CHANGELOG.md

```diff
@@ -1,7 +1,29 @@
 # Changelog
+## 1.32.5 (2024-11-24)
+### Breaking
+* Removed PA134 from the old validate. The validation ensures that the pack has exactly one category and that this category is valid. [#4673](https://github.com/demisto/demisto-sdk/pull/4673)
+
+### Feature
+* Modified PA103 validation (validate pack categories). The validation will now allow multiple categories for a single pack. [#4673](https://github.com/demisto/demisto-sdk/pull/4673)
+* Modified the configuration section generated by the generate-docs command by removing navigation steps and generalizing the code to accommodate different Cortex platforms. [#4585](https://github.com/demisto/demisto-sdk/pull/4585)
+* Added support for non-default registry ports when using custom docker images. i.e: 'registry:5000/repository/image:main' [#4674](https://github.com/demisto/demisto-sdk/pull/4674)
+* Added support for utilizing content_status.json within content pipelines. [#4656](https://github.com/demisto/demisto-sdk/pull/4656)
+
+### Fix
+* Fixed an issue where ***demisto-sdk pre-commit*** inadvertently skipped all files when executed with the *--all-files* flag. [#4668](https://github.com/demisto/demisto-sdk/pull/4668)
+* Fixed an issue where demisto-sdk commands would fail in some system setups. [#4666](https://github.com/demisto/demisto-sdk/pull/4666)
+* Fixed an issue where logging would fail when linking PR to Jira issue in the GitHub workflow step. [#4649](https://github.com/demisto/demisto-sdk/pull/4649)
+* Fixed an issue where logger crashed when imported as library. [#4657](https://github.com/demisto/demisto-sdk/pull/4657)
+* Fixed an issue where logging would fail when creating a Docker image. [#4648](https://github.com/demisto/demisto-sdk/pull/4648)
+
+### Internal
+* Moved RN106 and RN107 to the new validate format. These validations ensure there are no missing release notes. [#4643](https://github.com/demisto/demisto-sdk/pull/4643)
+* Fixed an issue where the git statuses of content objects related files were not calculated correctly. [#4643](https://github.com/demisto/demisto-sdk/pull/4643)
+
+
 ## 1.32.4 (2024-11-10)
 ### Breaking
-* The following commands will be deprecated and removed entirely in the next SDK release in two weeks: convert, create-content-artifacts, create-id-set, extract-code and lint.
+* The following commands will be deprecated and removed entirely in the next minor demisto-sdk release: convert, create-content-artifacts, create-id-set, extract-code and lint.
```

Comment:
* The following commands will be deprecated and removed entirely in the next SDK release in two weeks: convert, create-content-artifacts, create-id-set, extract-code and lint.
```suggestion
* The following commands will be deprecated and removed entirely in the next SDK release in two weeks: convert, create-content-artifacts, create-id-set, extract-code and lint.
```